migrations/20190117175650_ingredients.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('ingredients', function(tbl) {
+    tbl.increments('id');
+    tbl
+      .integer('recipe_id')
```

Comment:
Great job w/ your modeling in your Google sheets. The migrations really reflect that, great job on figuring out the foreign keys needed.